exporter/opentelemetry-exporter-prometheus-remote-write/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Changelog
+
+## Unreleased
+
+## Initial Release
```

Comment:
nit:
this change can just be under the `## Unreleased` section, when the release is done, the correct version tag title will be applied to the changelog

Reply:
Rebased and updated CHANGELOG to use Unreleased